pages/search.js

```diff
@@ -338,7 +340,7 @@ class Search extends React.Component {
           // When `hideFailure` is true, add `failure=false` in the query
           query[queryParam] = false
         } else {
-          delete query[queryParam]
+          query[queryParam] = true
```

Comment:
I think this change came from a different branch and should be included a separate PR.

Reply:
Okay let me change it.